src/styles/index.ts

```diff
@@ -2350,11 +2350,6 @@ const styles = (theme: ThemeColors) =>
             backgroundColor: theme.appBG,
         },
 
-        avatarLarge: {
-            width: variables.avatarSizeLarge,
-            height: variables.avatarSizeLarge,
-        },
```

Comment:
Was this change the only use of this?

Reply:
Yes, after switching the workspace / group icon size to be 100x100 pixels it turned out the previous style was only used there and this could be removed.